ftw/contentpage/profiles/default/jsregistry.xml

```diff
@@ -8,4 +8,9 @@
 
  <javascript cacheable="True" compression="safe" cookable="True"
     enabled="True" expression="" id="++resource++ftw.contentpage.resources/toggle_faq.js" inline="False"/>
+
+ <javascript cacheable="True" compression="safe" cookable="True"
```

Comment:
@shylux Please do not register this js globaly.  It's only used in very specific usecases (textBlock + right permission).
You can put it onto a AT filed as helper js.